content/en/docs/concepts/storage/volume-snapshot-classes.md

```diff
@@ -54,6 +54,16 @@ parameters:
 Volume snapshot classes have a snapshotter that determines what CSI volume plugin is
 used for provisioning VolumeSnapshots. This field must be specified.
 
+### VolumeSnapshotContent Deletion/Retain Policy
+
+Volume snapshots that are dynamically created by a volume snapshot class will have the
+snapshot content deletion policy specified in the `deletionPolicy` field of the class, which can be
+either `Delete` or `Retain`. If no `deletionPolicy` is specified when a
+`VolumeSnapshotClass` object is created, it will default to `Delete`.
+
+Volume snapshots that are created manually and managed via a volume snapshot class will have
+whatever deletion policy they were assigned at creation.
```

Comment:
I think if user does not specify the policy for manually created volume snapshot and content without a snapshot class will have default value of retain? but need to double check.

Reply:
never mind, I see your explanation below